shared/actions/people.js

```diff
@@ -92,9 +90,15 @@ const _skipTodo = (action: PeopleGen.SkipTodoPayload) => {
   ])
 }
 
-let _wasOnPeopleTab = false
+let _wasOnPeopleTab = true
```

Comment:
Assume that we start on the people tab more often than other tabs for refresh strategy